packages/react/src/components/MultiSelect/MultiSelect.stories.js

```diff
@@ -226,16 +226,24 @@ Playground.argTypes = {
 
 export const Default = () => {
   return (
-    <div style={{ width: 300 }}>
-      <MultiSelect
-        label="Multiselect Label"
-        id="carbon-multiselect-example"
-        titleText="Multiselect title"
-        helperText="This is helper text"
-        items={items}
-        itemToString={(item) => (item ? item.text : '')}
-        selectionFeedback="top-after-reopen"
-      />
+    <div
+      style={{
+        width: 2000,
+        height: 2000,
+        display: 'flex',
+        placeItems: 'center',
+      }}>
+      <div style={{ width: 300 }}>
+        <MultiSelect
+          label="Multiselect Label"
+          id="carbon-multiselect-example"
+          titleText="Multiselect title"
+          helperText="This is helper text"
+          items={items}
+          itemToString={(item) => (item ? item.text : '')}
+          selectionFeedback="top-after-reopen"
+        />
+      </div>
```

Comment:
I'd be in favor of a new `with autoAlign` story if you think that would be a good idea.

If you want to keep it on these stories, which is totally fine by me, we should call `scrollIntoView` so the component is immediately visible on render. I think there's an example of it in the popover autoAlign stories

Reply:
- Added `autoAlign` I forgot we are not exposing as default the FloatingUI for now.
- Added new stories called Experimnetal Auto Align
- Added changes to Playground

I don't think we have to copy the format to every story in this point. We can copy in every story once we have it as default.